recipes/r-rphylip/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = 'Rphylip' %}
+{% set version = '0.1-23' %}
+
+{% set posix = 'm2-' if win else '' %}
```

Comment:
everything referring to win can be excluded, we do not build for the Windows platform.